artifacts/scripts/publish_template.sh

```diff
@@ -89,6 +89,7 @@ EXTRA_ARGS=()
 PUSH_SCRIPT=../push-tags-${REPO}-${DST_BRANCH}.sh
 echo "#!/bin/bash" > ${PUSH_SCRIPT}
 chmod +x ${PUSH_SCRIPT}
-/sync-tags --upstream-remote upstream-kube --upstream-branch "${SRC_BRANCH}" \
+/sync-tags --repo-org kubernetes --repo-name kubernetes \
```

Comment:
@sttts so this needed to be updated to pass the repo-org and repo-name, but I guess this will be different for other repo/orgs and this template can be modified per-usage?

Reply:
It's just a constant. We can use a generic one like `upstream`. Just grep in util.sh for it.